lib/web_ui/lib/pointer.dart

```diff
@@ -153,7 +153,23 @@ class PointerData {
   }
 }
 
+/// A function used to notify the engine about how a [PointerDataPacket] was used in the framework.
+///
+/// For example, can be used to [preventDefault] the source events on the web.
+typedef _PointerDataPacketAcknowledgeFn = void Function({
+  bool? preventDefault,
+});
+
+// The default _PointerDataPacketAcknowledgeFn, which is a noop.
+void _defaultPointerDataPacketAcknowledgeFn ({
+  bool? preventDefault,
```

Comment:
Why is this nullable? What does it mean to pass in null?

Reply:
It means the user has no preference, so the platform should do whatever it deems the default behavior. I guess I could encode the default in the signature of the function; also this is a remnant from when we had other things here in addition to preventDefault :/